trading_funcs.py

```diff
@@ -187,9 +229,14 @@ def get_relavant_values(start_date: str, end_date: str, stock_symbol: str,
 
 def scale(num: float, data: List) -> float:
     """
-    Scales List between 0 and 1
+    Scale the number between the minimum and maximum values of the data
```

Comment:
This is misleading. It does scale between 0 and 1.

What about something like "Scales the list between  and 1 using the `min` and `max` values in the data"